app/models/concerns/planning_application/notification.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class PlanningApplication
+  module Notification
```

Comment:
It's just occurred to me...I think it would make sense for me to wait until you've merged this and then move [this](https://github.com/unboxed/bops/pull/707/files#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29bR39) into it!

Reply:
Is that not just related to the User model ?

Reply:
Oh doh, yes you're right...I'm jumping between too many things at once 🤦

Reply:
No worries haha 😄